### PR TITLE
[150] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by Kafabidemi**

### CODE REVIEW

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Remove the spacing in your file names

**Comment by Kafabidemi** [comment]
Done, Kindly Review

File: submissions/adeyemoKafayat/html-forms/contactform.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+			<option value="Rwanda">Rwanda</option>
+			<option value="Ghana">Ghana</option>
+			<option value="Sa">S/Africa</option>
+		</datalist><br><br>
+		<label for="comments">Write your comments and suggestions about our products</label><br>
+		<textarea name="comments" id="comments" cols="50" rows="10"></textarea><br>
+		<button onclick="alart ('Hello, and welcome to my website')">Say Hello</button><br>
+		<input type="image" name="image-button" src="cloth.jpg" width="200" height="50"/>
+		</fieldset>
+			</form>
```
Comment (Ireti203) [inline submissions/adeyemoKafayat/html-forms/contactform.html:63]: Kindly format properly

File: submissions/adeyemoKafayat/html-forms/contactform.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+</head>
+<body>
+	<h1>Product Survey Form</h1>
+	 <nav>
+        <ul>
+           <li><a href="../HTML/product.html">Product</a></li>
+           <li><a href="contact form.html">Contact Form</a></li>
+           <li><a href="contact us.html">Contact Us</a></li>
+            <li><a href="#">About Us</a></li>
+            <li><a href="#">Cart</a></li>
```
Comment (Ireti203) [inline submissions/adeyemoKafayat/html-forms/contactform.html:15]: Format properly

File: submissions/adeyemoKafayat/HTML/Product.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    <title>Halal Wears</title>
+</head>
+<body>
+    <h1>Halal Wears</h1>
+    <nav>
+        <ul>
+            <li><a href="product.html">Product</a></li>
+           <li><a href="../html-forms/contact form.html">Contact Us</a></li>
+            <li><a href="#">About Us</a></li>
+            <li><a href="#">Cart</a></li>
```
Comment (Ireti203) [inline submissions/adeyemoKafayat/HTML/Product.html:15]: Format properly

**Review by Ireti203 — COMMENTED** [review]
Kindly rename the Product.html file as product.html

**Review by Ireti203 — CHANGES_REQUESTED** [review]
This task ought to include only the navigations added to your page. You already have the product.html file in your folder before, you dont have to create another.

**Comment by Kafabidemi** [comment]
Changes made, Kindly review

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**